Libraries/LibWeb/Encoding/TextEncoder.cpp

```diff
@@ -53,6 +53,11 @@ GC::Ref<JS::Uint8Array> TextEncoder::encode(String const& input) const
 // https://encoding.spec.whatwg.org/#dom-textencoder-encodeinto
 TextEncoderEncodeIntoResult TextEncoder::encode_into(String const& source, GC::Root<WebIDL::BufferSource> const& destination) const
 {
+    // AD-HOC: Return early if destination is detached. This is not explicitly handled in the spec,
+    //         however no bytes are copied as destinations size is always zero in this case.
+    if (destination->viewed_array_buffer()->is_detached())
+        return { 0, 0 };
```

Comment:
There is an existing spec bug about this, see:
https://github.com/whatwg/encoding/issues/324

We should link that issue in the code comment here.

Reply:
Should we go ahead and tag that issue with agenda+ for the next WHATNOT meeting for the spec editors? Seems like it got a decision stalled. If nothing comes of that we could probably draft the spec prose ourselves

Reply:
I think the proposed solution in the issue of throwing sounds good, although all other engines do nothing instead. How can I help to restart the discussion and propose changes to the spec?

Reply:
I'd start by leaving a comment that issue - when they suggested throwing an exception, it seems the API was still new and unlikely to have see real world use. That's probably no longer the case and they may instead chose to match web reality of returning 0.